### PR TITLE
Add commercial_partner_id on supplier invoice and

### PR DESCRIPTION
**Pull request by vrenaville**
set it with on_change_partner_id in order to have
company bank account in the related field selection

### CODE REVIEW

**Comment by vrenaville** [comment]
@nbessi Could you please review my change

File: l10n_ch_base_bank/invoice.py [hunk 1]
```diff
@@ -46,8 +46,21 @@ def onchange_partner_id(self, cursor, uid, ids, invoice_type, partner_id,
                     partner_id,
                     context
                 )
+                if p.commercial_partner_id:
+                    """
+                    We set the commercial_partner_id in the view
+                    to be able to filter in the view in the field
+                    partner_bank_id
+                    """
+                    res['value']['commercial_partner_id'] = p.commercial_partner_id.id
```
Comment (nbessi) [inline l10n_ch_base_bank/invoice.py]: Please add an intermediate variable

**Comment by nbessi** [comment]
One pep8 issue but else LGTM. Please can you create a issue for the record

**Comment by vrenaville** [comment]
@nbessi  thanks for the review I have just fixed it

**Comment by mdietrichc2c** [comment]
:+1:

**Comment by nbessi** [comment]
If it is green :+1:

**Comment by fclementic2c** [comment]
Does not work if the supplier invoice is created from a PO.
Otherwise, it is Ok

**Comment by vrenaville** [comment]
@fclementic2c The purchase order doen't call onchange partner, but with my fix the user can select bank account from the commercial_partner_id, so it work, but it's need a manual operation.
It can be an improvement but it's not a side effect of my PR

**Comment by fclementic2c** [comment]
ok then :+1:

**Pull request merged**